src/opentimelineio/stringUtils.h

```diff
@@ -11,7 +11,6 @@ namespace opentimelineio { namespace OPENTIMELINEIO_VERSION  {
     
 void fatal_error(std::string const& errMsg);
 
-std::string demangled_type_name(const char* name);
```

Comment:
Does this also need to be bracketed by `#if defined(__GNUC__) or defined(__clang__)` as in the .cpp file?

Reply:
...Oh I see that version became private instead of public.  We might want to change the name in that case so that its clear that that overloaded version of `demangled_type_name` is for wrapping in the `GNUC`/`__clang__`  environment.